examples/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
+    # Settings when 'examples' is the root projet
+    cmake_minimum_required(VERSION 3.20)
+    project(zenohcxx_examples LANGUAGES C CXX)
+    include(../cmake/helpers.cmake)
+    set_default_build_type(Release)
+    configure_include_project(ZENOHCXX_ZENOHC zenohc zenohc::lib "../../zenoh-c" zenohc "https://github.com/eclipse-zenoh/zenoh-c" "")
+    configure_include_project(ZENOHPICO zenohpico zenohpico "../../zenoh-pico" zenohc "https://github.com/eclipse-zenoh/zenoh-pico" "")
```

Comment:
should we defined it as `ZENOHCXX_ZENOHPICO` to be aligned with the `ZENOHCXX_ZENOHC`?

Reply:
Yes, you are right. I use ZENOHCXX_ZENOHC prefix here to avoid possible conflicts with settings of own zenohc settings, which have ZENOHC prefix. I need to to same for zenohpico (and also it's better to make zenohpico to use settings with ZENOHC prefix)